bigquery_etl/schema/__init__.py

```diff
@@ -1,16 +1,16 @@
 """Query schema."""
 
-from google.cloud import bigquery
 import json
+import os
 from pathlib import Path
 from tempfile import NamedTemporaryFile
-from typing import Any, Dict, Optional, List
+from typing import Any, Dict, List, Optional
 
 import attr
-import os
 import yaml
+from google.cloud import bigquery
 
-from bigquery_etl.dryrun import DryRun
+from .. import dryrun
```

Comment:
this allows `bigquery_etl/dryrun.py` to do `from .schema import SCHEMA_FILE, Schema` without a circular import exception.